spec/ysh-proc.test.sh

```diff
@@ -238,7 +238,7 @@ p
 ## STDOUT:
 ## END
 
-#### procs are in same namespace as shell functions
+#### declare -F prints procs and shell-funcs
```

Comment:
I wonder if we should say that `declare -F` is the "old OSH shell world" and have it only print shell functions

But we can think about that, this is great progress

Reply:
Good point. I guess we could use `pp proc`... but I'll leave it as `declare -F` for now.